roles/openshift_node/tasks/config.yml

```diff
@@ -2,10 +2,6 @@
 - name: Install the systemd units
   import_tasks: systemd_units.yml
 
-- name: Install Node system container
```

Comment:
This is already done in systemd_units, which is called just above and also in upgrades.